### PR TITLE
Includes shims

### LINKED ISSUE #17: Does not work in Node < 8
`jest-tap-reporter` uses `.padEnd()` and `.padStart()` methods available since Node 8, so in versions before 8 it fails.

Fix: shim those methods.

Also test on Travis on Node versions 6 and 7.

### PR DESCRIPTION
**Pull request by goatslacker**
Includes the shims for the environments that don't have padStart and padEnd on String.prototype

Fixes #17

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
 /* eslint-disable filenames/match-exported */
+
+const padEnd = require('string.prototype.padend');
+const padStart = require('string.prototype.padstart');
 const TapReporter = require('./src/TapReporter');
 
+padEnd.shim();
+padStart.shim();
```
Comment (ljharb) [inline index.js]: Not a huge deal but it’d be better if this used it as a function instead of modifying the environment.

File: package.json [hunk 2]
```diff
@@ -38,17 +38,22 @@
...
+          "showInternalStackTraces": false
+        }
+      ]
     ],
     "testRegex": "(test|src)\\/.+\\.(test|spec)\\.jsx?$"
   },
   "dependencies": {
+    "@babel/code-frame": "7.0.0-beta.34",
     "chalk": "^2.3.0",
+    "string.prototype.padend": "^3.0.0",
```
Comment (goatslacker) [inline package.json]: I ran `npm i` with `--save` so it re-ordered these imports and changed the ordering of your reporters config above. I can revert this back if you'd like.

File: index.js (hunk 1 above)
Comment (goatslacker) [reply index.js]: I agree, I left the usage as-is but I also think we should switch to use as a function.

Comment (streamich) [reply index.js]: @goatslacker can you please use functions instead of shims?

**Comment by streamich** [comment]
Will merge it here: https://github.com/MailOnline/jest-tap-reporter/pull/43

**Pull request closed without merging**